default/python/AI/FleetUtilsAI.py

```diff
@@ -26,7 +26,7 @@ def stats_meet_reqs(stats, requirements):
     :rtype: bool
     """
     for key in requirements:
-        if stats.get(key, 0) < requirements[key]:
+        if key not in stats or stats.get(key, 0) < requirements[key]:
```

Comment:
```TypeError: '<' not supported between instances of 'int' and 'TargetSystem'``` sounds like an underlying issue with the code which should not be silenced like this.

Reply:
`target_system` key contains `TargetSystem` type. I suppose in python2 `<` returned `False` for such comparsion.

Reply:
Split condition to two separate ifs.  First one  `if key not in stats: # skip requirements not related to stats`

Reply:
@Cjkjvfnby Should I use `continue` in this case?

Reply:
```requirements``` should logically be a subset of ```stats``` and ```target_system``` should not be in the dict but passed around differently. This is beyond the scope of this PR though.

For now, we should probably log a warning.
```
if key not in stats:
    if key != "target_system":  # expected not to be in stats
        warn("Requirement %s not in stats", key)
    continue
```

Reply:
Fixed.